### PR TITLE
Notification Log tab page

### PR DESCRIPTION
**Pull request by Georgi2704**
Added the page for the Notification Log tab in the service ticket details.

### CODE REVIEW

File: packages/orchestrator-ui-components/src/utils/date.ts [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import { Locale } from '../types/types';
```
Comment (Georgi2704) [inline packages/orchestrator-ui-components/src/utils/date.ts:1]: A better naming of the types folder or the files inside could be proposed, so they don't clash.

File: apps/wfo-ui-surf/components/WfoAvatar/styles.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        alignItems: 'center',
+        display: 'flex',
+        justifyContent: 'center',
+        borderRadius: '24px',
+        border: `2px solid ${theme.colors.emptyShade}`,
+    };
+
+    const openIconStyle = css({
+        ...stepStateIcon,
+        backgroundColor: '#D9EDE1',
```
Comment (DutchBen) [inline apps/wfo-ui-surf/components/WfoAvatar/styles.ts:17]: Are these colors not part of the theme? This is surf specific but I would still expect everything to follow the general design

File: packages/orchestrator-ui-components/src/icons/WfoBell.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+export const WfoBell: FC<WfoIconProps> = ({ width = 20, height = 20 }) => (
+    <svg
+        width={width}
+        height={height}
+        viewBox="0 0 24 24"
+        version="1.1"
+        xmlns="http://www.w3.org/2000/svg"
+    >
+        <title>icon/contact-envelope-fill</title>
```
Comment (DutchBen) [inline packages/orchestrator-ui-components/src/icons/WfoBell.tsx:13]: This should be icon/bell probably

File: packages/orchestrator-ui-components/src/utils/date.ts (hunk 1 above)
Comment (DutchBen) [reply packages/orchestrator-ui-components/src/utils/date.ts:1]: I have no better suggestions :(. Are you sure you can't use the Locale from the package types?

**Review by DutchBen — APPROVED** [review]

File: packages/orchestrator-ui-components/src/utils/date.ts (hunk 1 above)
Comment (Georgi2704) [reply packages/orchestrator-ui-components/src/utils/date.ts:1]: Yep, it fails the unit-tests if use '@/types'

File: apps/wfo-ui-surf/components/WfoAvatar/styles.ts (hunk 2 above)
Comment (Georgi2704) [reply apps/wfo-ui-surf/components/WfoAvatar/styles.ts:17]: These colors are also used in the WfoWorkflowSteps and they are also not part of the theme.

File: packages/orchestrator-ui-components/src/icons/WfoBell.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import React, { FC } from 'react';
+
+import { WfoIconProps } from './WfoIconProps';
+
+export const WfoBell: FC<WfoIconProps> = ({ width = 20, height = 20 }) => (
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/icons/WfoBell.tsx:5]: Other icons in this folder are default 24 px, suggesting to do the same here

File: apps/wfo-ui-surf/translations/en-GB.json [hunk 5]
```diff
@@ -68,6 +68,11 @@
                             "extraInformation": "Extra info",
                             "impact": "Impact"
                         }
+                    },
+                    "notificationLog": {
+                        "nl": "NL",
+                        "en": "EN",
+                        "noLogsAvailableForThisServiceTicket": "No logs available for this service ticket"
```
Comment (ricardovdheijden) [inline apps/wfo-ui-surf/translations/en-GB.json:75]: Did you add tje Dutch translations?

File: apps/wfo-ui-surf/components/WfoAvatar/styles.ts [hunk 6]
```diff
@@ -0,0 +1,38 @@
+import { EuiThemeComputed } from '@elastic/eui/src/services/theme/types';
+import { css } from '@emotion/react';
+
+export const getStyles = (theme: EuiThemeComputed) => {
```
Comment (ricardovdheijden) [inline apps/wfo-ui-surf/components/WfoAvatar/styles.ts:4]: In addition to the discussion below i would like to suggest to replace all the magic numbers/colors with a value from the theme. Or at least as much as possible

File: apps/wfo-ui-surf/components/WfoAvatar/styles.ts (hunk 2 above)
Comment (wouter1975) [reply apps/wfo-ui-surf/components/WfoAvatar/styles.ts:17]: would be good to use the theme, maybe also fix this in other parts of the code?

**Review by wouter1975 — APPROVED** [review]

**Pull request merged**